koku/api/settings/tags/mapping/view.py

```diff
@@ -0,0 +1,144 @@
+        serializer = self.serializer_class(filtered_qset, many=True)
+        paginator = ListPaginator(serializer.data, request)
+        response = paginator.paginated_response
+        response = format_tag_mapping_relationship(response)
+
+        return response
+
+
+class SettingsTagMappingChildView(generics.GenericAPIView):
+    queryset = (
+        EnabledTagKeys.objects.exclude(parent__isnull=False).exclude(child__parent__isnull=False).filter(enabled=True)
+    )
+    serializer_class = EnabledTagKeysSerializer
+    permission_classes = (SettingsAccessPermission,)
+    filter_backends = (DjangoFilterBackend,)
+    filterset_class = SettingsEnabledTagKeysFilter
+
+    @method_decorator(never_cache)
+    def get(self, request: Request, **kwargs):
+        filtered_qset = self.filter_queryset(self.get_queryset())
+        serializer = self.serializer_class(filtered_qset, many=True)
+        paginator = ListPaginator(serializer.data, request)
+        response = paginator.paginated_response
+
+        return response
+
+
+class SettingsTagMappingParentView(generics.GenericAPIView):
+    queryset = EnabledTagKeys.objects.exclude(child__parent__isnull=False).filter(enabled=True)
+    serializer_class = EnabledTagKeysSerializer
+    permission_classes = (SettingsAccessPermission,)
+    filter_backends = (DjangoFilterBackend,)
+    filterset_class = SettingsEnabledTagKeysFilter
+
+    @method_decorator(never_cache)
+    def get(self, request: Request, **kwargs):
+        filtered_qset = self.filter_queryset(self.get_queryset())
+        serializer = self.serializer_class(filtered_qset, many=True)
+        paginator = ListPaginator(serializer.data, request)
+        response = paginator.paginated_response
+
+        return response
+
+
+class SettingsTagMappingChildAddView(APIView):
+    permission_classes = (SettingsAccessPermission,)
+
+    def put(self, request):
+        serializer = AddChildSerializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        parent_row = EnabledTagKeys.objects.get(uuid=serializer.data.get("parent"))
+        children = EnabledTagKeys.objects.filter(uuid__in=serializer.data.get("children"))
+        tag_mappings = [TagMapping(parent=parent_row, child=child_row) for child_row in children]
+        TagMapping.objects.bulk_create(tag_mappings)
+        return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class SettingsTagMappingChildRemoveView(APIView):
+    permission_classes = (SettingsAccessPermission,)
+
+    def put(self, request: Request):
+        children_uuids = request.data.get("ids", [])
+        if not EnabledTagKeys.objects.filter(uuid__in=children_uuids).exists():
+            return Response({"detail": "Invalid children UUIDs."}, status=status.HTTP_400_BAD_REQUEST)
+
+        TagMapping.objects.filter(child__in=children_uuids).delete()
+
+        return Response({"detail": "Children deleted successfully."}, status=status.HTTP_204_NO_CONTENT)
```

Comment:
```suggestion
        return Response(status=status.HTTP_204_NO_CONTENT)
```
One last thing! This should just be a 204 status with no message.